frontend/public/components/about-modal.tsx

```diff
@@ -116,8 +121,9 @@ export const AboutModal: React.FC<AboutModalProps> = (props) => {
     >
       {!customBranding && (
         <p>
-          OpenShift is Red Hat&apos;s container application platform that allows developers to
-          quickly develop, host, and scale applications in a cloud environment.
+          {t(
+            'modal~OpenShift is Red Hat&apos;s container application platform that allows developers to quickly develop, host, and scale applications in a cloud environment.',
```

Comment:
You don't want to use the `&apos;` entity here.

```suggestion
            'modal~OpenShift is Red Hat\'s container application platform that allows developers to quickly develop, host, and scale applications in a cloud environment.',
```